lib/components/normal-components/Jumper.ts

```diff
@@ -77,12 +77,10 @@ export class Jumper<PinLabels extends string = never> extends NormalComponent<
     })
     const pinLabelToPortId: Record<string, string> = {}
     // Map pin labels ("1", "2", etc.) to pcb_port_id
-    for (const port of pcb_ports) {
-      const match = port.source_port_id && port.source_port_id.match(/(\d+)$/)
-      if (match) {
-        const label = (parseInt(match[1], 10) + 1).toString()
-        pinLabelToPortId[label] = port.pcb_port_id
-      }
+    for (let i = 0; i < pcb_ports.length; i++) {
+      const port = pcb_ports[i]
```

Comment:
Use the port hints or the literal label value, the order isnt guaranteed _by the api contract_, it doesnt matter if it works now, it _will_ break if the api contract doesn't specifically guarantee the order